state-sync/diem-data-client/src/diemnet/mod.rs

```diff
@@ -434,17 +482,28 @@ impl DataSummaryPoller {
             let storage_summary = match result {
                 Ok(storage_summary) => storage_summary,
                 Err(error) => {
-                    trace!(
-                        "Error encountered when polling peer ({:?})! Error: {:?}",
-                        peer,
-                        error
+                    error!(
+                        (LogSchema::new(LogEntry::StorageSummaryResponse)
+                            .event(LogEvent::PeerPollingError)
+                            .message(&format!("Error encountered when polling peer {:?}", peer))
```

Comment:
We should probably log the `peer` as the same way as above, right? e.g., 
```
...
.message(....)
.peer(peer)
...
```

Reply:
Used `.peer` field.